projects/plugins/vaultpress/to-test.md

```diff
@@ -1,22 +0,0 @@
-# 2.0
```

Comment:
You could keep this if you want (but probably production-exclude it). In #20137 I'm moving Jetpack's to-test.md into projects/plugins/jetpack and adjusting the build so having this file will do the right thing with the Beta plugin.

Reply:
I think it's fine to get rid of it for now. The VaultPress team hasn't relied on it for their past few releases.